http/grpc/src/test/java/io/quarkus/ts/http/grpc/SeparateServerIT.java

```diff
@@ -20,8 +24,17 @@ public CloseableManagedChannel getChannel() {
     }
 
     @Override
-    public RestService app() {
-        return app;
+    public WebClient getWebClient() {
+        if (webClient == null) {
+            webClient = app.mutiny();
+        }
+        return webClient;
     }
 
+    @AfterAll
+    static void afterAll() {
+        if (webClient != null) {
+            webClient.close();
```

Comment:
yep; I only implemented `mutinyHttps` and that one I implemented according to my liking (close Vert.X, close WebClient), but I didn't refactor the original method. if it is any consolation, I remember when I was implementing `mutinyHttps`, the ordinary ones weren't even closed in our FW. I don't know if it changed since then, as I have habit to change thing.

Few moments ago I have opened: https://github.com/quarkus-qe/quarkus-test-framework/issues/1284